external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaContinuousSourceSuite.scala

```diff
@@ -19,15 +19,24 @@ package org.apache.spark.sql.kafka010
 
 import org.apache.kafka.clients.producer.ProducerRecord
 
+import org.apache.spark.SparkContext
 import org.apache.spark.sql.Dataset
 import org.apache.spark.sql.execution.datasources.v2.ContinuousScanExec
 import org.apache.spark.sql.execution.streaming.ContinuousTrigger
 import org.apache.spark.sql.streaming.Trigger
+import org.apache.spark.sql.test.TestSparkSession
 
 // Run tests in KafkaSourceSuiteBase in continuous execution mode.
 class KafkaContinuousSourceSuite extends KafkaSourceSuiteBase with KafkaContinuousTest {
   import testImplicits._
 
+  override protected def createSparkSession = new TestSparkSession(
```

Comment:
Why we need to set `spark.executor.cores` in both here and https://github.com/apache/spark/pull/30175/files#diff-d56b5958c6efb7fe729a960641a907b142f0ba0268ec2e31f95e1e42044244e4R43?

Reply:
These 2 places don't have common inheritance root. So I have to set this separately.